api/client/hijack.go

```diff
@@ -19,7 +33,11 @@ func (cli *DockerCli) holdHijackedConnection(tty bool, inputStream io.ReadCloser
 			} else {
 				_, err = stdcopy.StdCopy(outputStream, errorStream, resp.Reader)
 			}
+			cli.restoreTerminal(inputStream)
+			restored = true
+
 			logrus.Debugf("[hijack] End of stdout")
+			logrus.Debugf("Hello world, this is a test")
```

Comment:
This line is just for testing for other ones review and test this PR, will be removed once it's ready to merge